misk/src/main/kotlin/misk/web/interceptors/RequestLoggingInterceptor.kt

```diff
@@ -144,3 +148,45 @@ class RequestLoggingInterceptor internal constructor(
     ) { builder.toString() }
   }
 }
+
+/**
+ * A set of per-action logging config overrides.
+ */
+data class RequestLoggingConfig(val actions: Map<String, ActionLoggingConfig>) {
+}
+
+/**
+ * This class should have all the same config options as [LogRequestResponse]. See that class for details.
+ */
+data class ActionLoggingConfig(
+  val ratePerSecond: Long = 10,
+  val errorRatePerSecond: Long = 0,
+  val bodySampling: Double = 0.0,
+  val errorBodySampling: Double = 0.0,
+  val excludedEnvironments: List<String> = listOf(),
+) {
+  companion object {
+    fun fromAnnotation(logRequestResponse: LogRequestResponse): ActionLoggingConfig = ActionLoggingConfig(
```

Comment:
`fun LogRequestResponse.toConfig() = ActionLogginConfig(...)` could be a bit more idiomatic Kotlin